packages/tools/src/stateManagement/annotation/config/ToolStyle.ts

```diff
@@ -35,6 +35,7 @@ class ToolStyle {
       lineWidth: '1',
       lineDash: '',
       shadow: true,
+      textBoxVisibility: '1',
```

Comment:
This should be a boolean

Reply:
similar to texBoxShadow below

Reply:
So I have to change '0' to 'false' for everyone too?

```typescript
const options = this.getLinkedTextBoxStyle(styleSpecifier, annotation);
if (options.visibility === '0') { // If changed to false
  return;
}
```

Reply:
I guess you should then say 

```
if (!options.visibility)
  return;
}
```
no?

Reply:
`!options.visibility` or `options.visibility === undefined`?

Reply:
I'm going to make a new PR